### ISSUE TITLE
How to run test in parallel? Can't find the proper configuration

### ISSUE THREAD

**Issue by bitliner** [angular/protractor#2516]
Hi,

I need to run o set of test on a single browser.

To speed up the process, I would like to shard the set of test among multiple browser in parallel.

One detail is that every test is on 2 concurrent browser, so before running the test I run a fork to create a new browser that represents a concurrent user.

My configuration follows

```
// ... other protractor parameters ...
capabilities: {
		browserName: 'firefox',
		// count: 2,
		shardTestFiles: true,
		maxInstances: 2,
	},
// ... other protractor parameters ...
```

I am testing multiple configuration settings, but I always get same result: test are not run in parallel, but every browser runs the same test. It is not sharding, but replication.

I'm trying to find a configuration based on https://github.com/angular/protractor/blob/master/docs/referenceConf.js but still I can't find any way to run test in parallel.

Any help?

**Comment by stalniy** [comment]
Configure Multicapabilities

**Comment by juliemr** [comment]
Do you have more than one file for your test scripts? `shardTestFiles` can only separate by file, not by individual test case. Otherwise, what you're doing should work.

**Comment by juliemr** [comment]
I'm aggressively closing issues that we are unable to reproduce, are support issues specific to one test, or deal with integrations with other projects that we don't directly support. If you feel you have a question, please see https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#questions. If you still have a bug report or feature request, please open a new issue. Thank you!
